mmv1/products/bigquery/terraform.yaml

```diff
@@ -95,6 +109,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       friendlyName: !ruby/object:Overrides::Terraform::PropertyOverride
         send_empty_value: true
+      isCaseInsensitive: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      defaultCollation: !ruby/object:Overrides::Terraform::PropertyOverride
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validateDefaultCollation'
```

Comment:
I would prefer we refrain from adding clientside validation here. If we expect values to be added over time on the api side this resource would then fail clientside. 

We can omit the validation and allow the serverside validation to be sufficient.

Reply:
> I would prefer we refrain from adding clientside validation here. If we expect values to be added over time on the api side this resource would then fail clientside.
> 
> We can omit the validation and allow the serverside validation to be sufficient.

@ScottSuarez, thank you, it makes sense to me. I removed client-side validation.